tera-templates/crate/details.html

```diff
@@ -0,0 +1,303 @@
+                                                    <i class="fa fa-fw fa-warning"></i> {{ release.version }}
+                                                </a>
+
+                                            {# If the release wasn't yanked and built correctly, display it normally #}
+                                            {%- elif release.build_status -%}
+                                                <a href="/crate/{{ details.name }}/{{ release.version }}" class="pure-menu-link">
+                                                    {{ release.version }}
+                                                </a>
+
+                                            {# If the release wasn't yanked but failed to build, show a warning #}
+                                            {%- else -%}
+                                                <a href="/crate/{{ details.name }}/{{ release.version }}" class="pure-menu-link warn"
+                                                    title="docs.rs failed to build {{ details.name }}-{{ release.version }}">
+                                                    <i class="fa fa-fw fa-warning"></i> {{ release.version }}
+                                                </a>
+                                            {%- endif -%}
+                                        </li>
+                                    {%- endfor -%}
+                                </ul>
+                            </div>
+                        </li>
+
+                        {# Display the crate owner's profile picture and a link to their docs.rs profile #}
+                        <li class="pure-menu-heading">Owners</li>
+                        <li class="pure-menu-item">
+                            {%- for owner in details.owners -%}
+                                <a href="/releases/@{{ owner[0] }}">
+                                    <img src="{{ owner[1] }}" alt="{{ owner[0] }}" class="owner">
+                                </a>
+                            {%- endfor -%}
+                        </li>
+                    </ul>
+                </div>
+            </div>
+
+            <div class="pure-u-1 pure-u-sm-17-24 pure-u-md-19-24 package-details" id="main">
+                {# If the release is not a library #}
+                {%- if not details.is_library -%}
+                    <div class="warning">
+                        {{ details.name }}-{{ details.version }} is not a library.
+                    </div>
+
+                {# If the release has been yanked and is a library #}
+                {%- elif details.yanked -%}
+                    <div class="warning">
+                        {{ details.name }}-{{ details.version }} has been yanked.
+                    </div>
+
+                {# If the build succeeded, isn't yanked and is a library #}
+                {%- elif details.build_status -%}
```

Comment:
Why is this an else? If it failed to build and also was yanked, don't we want to show both warnings?

Reply:
This is an exact recreation of the previous behavior, I don't know if multiple `warning`s will conflict in some way (CSS wise)